test/web-platform-tests/to-run.yaml

```diff
@@ -169,7 +169,7 @@ Element-childElementCount-dynamic-add-xhtml.xhtml: [fail, Unknown]
 Element-childElementCount-dynamic-remove-xhtml.xhtml: [fail, Unknown]
 Element-childElementCount-nochild-xhtml.xhtml: [fail, Unknown]
 Element-childElementCount-xhtml.xhtml: [fail, Unknown]
-Element-closest.html: [fail, Not implemented]
+Element-closest.html: [fail, Unknown]
```

Comment:
Failed in "Element.closest with context node 'test11' and selector ':invalid'":

Reply:
@dperini @Zirro I thought we supported `div > :scope` now?

Reply:
@domenic `:scope` is supported in general, but the `div > :scope` test hits a bug in the implementation. It was not fixed earlier due to being confused with the last test which fails in all browsers - see https://github.com/web-platform-tests/wpt/issues/11316.

I've spoken to @dperini and it should be fixed in the next release of nwsapi.

Reply:
Ah OK. @caub, we can either sit tight and wait for the next nwsapi release, or we can copy over the test and comment out the failing one for now---see other "dont-upstream.html" examples.